Strings/is_ palindrome.js

```diff
@@ -0,0 +1,21 @@
+// Check whether a given string is a Valid Palindrome in Javascript
```

Comment:
Could you add this solution using Two pointers approach as well ?
check this out https://github.com/akgmage/data-structures-and-algorithms/blob/main/Strings/is_pallindrome.go